packages/augur-ui/src/modules/portfolio/components/positions/positions.tsx

```diff
@@ -70,12 +69,13 @@ interface PositionsState {
 }
 
 export default class Positions extends Component<PositionsProps, PositionsState> {
+  state: {
```

Comment:
this should be `state = {` otherwise it fails to render.

Reply:
Fixed initial state bug.